### PR TITLE
[Ready] Makes paper work for cargo to do for spare credits!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds in Completed Paper work worth 50 credits to sell back to NT
Adds in Incomplete Paper work witch if you try to sell back will cost you 500 credits!
Adds in a new crafting recipe that will make one Completed paper work but costs 2 incomplete paper work and a fountain pen
Brand new crate holding the goodies to make 5 sets of Completed Paper work with two fountains pens to spare!

[why]
Realism.
Spare way for cargo to get small amounts of quick cash for legit doing there job.

### CODE REVIEW

**Comment by Ragolution** [comment]
Why a fountain pen?

File: code/modules/cargo/exports/manifest.dm [hunk 1]
```diff
@@ -76,3 +76,17 @@
...
+/datum/export/paperwork_correct
+	cost = -50
+	unit_name = "correct paperwork"
+	export_types = list(/obj/item/paper/fluff/jobs/cargo/manifest/paperwork_correct)
+
+// Paper work retruned
+
+/datum/export/paperwork_correct
+	cost = 500 // Failed to meet NT standers
+	unit_name = "retruned late paperwork"
```
Comment (Ragolution) [inline code/modules/cargo/exports/manifest.dm]:
```suggestion
	unit_name = "returned incorrect paperwork"
```

File: code/modules/cargo/order.dm [hunk 2]
```diff
@@ -95,3 +95,14 @@
 			while(--lost >= 0)
 				qdel(pick(C.contents))
 	return C
+
+//Paperwork for NT
+/obj/item/paper/fluff/jobs/cargo/manifest/paperwork
+	name = "Undone Paper work"
```
Comment (Ragolution) [inline code/modules/cargo/order.dm]:
```suggestion
	name = "Incomplete Paperwork"
```

File: code/modules/cargo/order.dm [hunk 3]
```diff
@@ -95,3 +95,14 @@
 			while(--lost >= 0)
 				qdel(pick(C.contents))
 	return C
+
+//Paperwork for NT
+/obj/item/paper/fluff/jobs/cargo/manifest/paperwork
+	name = "Undone Paper work"
+	desc = "Undone paper work that has been left undone for freelancers to complete."
```
Comment (Ragolution) [inline code/modules/cargo/order.dm]:
```suggestion
	desc = "These should've been filled out four months ago! Unfinished grant papers issued by Nanotrasen's finance department. Complete this page for additional funding."
```

File: code/modules/cargo/order.dm [hunk 4]
```diff
@@ -95,3 +95,14 @@
...
 	return C
+
+//Paperwork for NT
+/obj/item/paper/fluff/jobs/cargo/manifest/paperwork
+	name = "Undone Paper work"
+	desc = "Undone paper work that has been left undone for freelancers to complete."
+	icon = 'icons/obj/bureaucracy.dmi'
+
+/obj/item/paper/fluff/jobs/cargo/manifest/paperwork_correct
+	name = "Done Paper work"
```
Comment (Ragolution) [inline code/modules/cargo/order.dm]:
```suggestion
	name = "Finished Paperwork"
```

File: code/modules/cargo/order.dm [hunk 5]
```diff
@@ -95,3 +95,14 @@
...
+	desc = "Paper work that has been completed, has a nice pen attached to the papers making sure they don't become messy well in travel."
```
Comment (Ragolution) [inline code/modules/cargo/order.dm]:
```suggestion
	desc = "A neat stack of filled-out forms, in triplicate and signed. Is there anything more satisfying? Make sure they get stamped."
```
Would be nice if we could do this with a clipboard, instead, honestly.

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -2412,6 +2412,22 @@
...
+	contains = list(/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/storage/box/fountainpens) // Two spare pens just in case
```
Comment (Ragolution) [inline code/modules/cargo/packs.dm:2429]:
```suggestion

```

File: code/modules/crafting/recipes.dm [hunk 7]
```diff
@@ -694,10 +694,18 @@
 					/obj/item/reagent_containers/food/snacks/grown/potato = 1)
 	category = CAT_MISC
 
+/datum/crafting_recipe/paperwork
+	name = "Filed Paper Work"
+	result =  /obj/item/paper/fluff/jobs/cargo/manifest/paperwork_correct
+	time = 90 //Takes time for people to file and complete paper work!
+	reqs = list(/obj/item/pen/fountain = 1,
```
Comment (Ragolution) [inline code/modules/crafting/recipes.dm]:
```suggestion
	reqs = list(/obj/item/clipboard = 1,
```

File: code/modules/cargo/exports/manifest.dm [hunk 8]
```diff
@@ -76,3 +76,17 @@
...
+// Paper work done correctly
+
+/datum/export/paperwork_correct
+	cost = -50
+	unit_name = "correct paperwork"
+	export_types = list(/obj/item/paper/fluff/jobs/cargo/manifest/paperwork_correct)
+
+// Paper work retruned
+
+/datum/export/paperwork_correct
```
Comment (Ragolution) [inline code/modules/cargo/exports/manifest.dm]:
```suggestion
/datum/export/paperwork_incorrect
```

Comment (Ragolution) [reply code/modules/cargo/exports/manifest.dm]: I really hope you tested this. :(

File: code/modules/cargo/packs.dm [hunk 9]
```diff
@@ -2412,6 +2412,22 @@
 	crate_name = "deluxe keg"
 	crate_type = /obj/structure/closet/crate
 
+/datum/supply_pack/misc/paper_work
+	name = "Freelance Paper work"
+	desc = "Need a little more credits for that bike for little Timmy? Well I'm willing to pay ya for some accounting that needs to get done. I'll send ya umm ten copies of some paper work and a fountain pens. I want my pens back, send them with the papers or don't get paid! There is two more pens in the box then you will need just in case, keep em or don't."
```
Comment (Ragolution) [inline code/modules/cargo/packs.dm]:
```suggestion
	desc = "The Nanotrasen Primary Bureaucratic Database Intelligence (PDBI) reports that the station has not completed its funding and grant paperwork this solar cycle. In order to gain further funding, your station is required to fill out (10) ten of these forms or no additional capital will be disbursed. We have sent you ten copies of the following form and we expect every one to be up to Nanotrasen Standards." // Disbursement. It's not a typo, look it up.
```

File: code/modules/cargo/packs.dm [hunk 10]
```diff
@@ -2412,6 +2412,22 @@
...
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork,
+					/obj/item/storage/box/fountainpens) // Two spare pens just in case
+	crate_name = "paper work"
```
Comment (Ragolution) [inline code/modules/cargo/packs.dm]:
```suggestion
	crate_name = "Paperwork"
```

File: code/modules/crafting/recipes.dm [hunk 11]
```diff
@@ -694,10 +694,18 @@
...
+					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork = 2)
```
Comment (Ragolution) [inline code/modules/crafting/recipes.dm:702]:
```suggestion
					/obj/item/paper/fluff/jobs/cargo/manifest/paperwork = 2, /obj/item/pen = 1)
```

**Review by Ragolution — CHANGES_REQUESTED** [review]
Seems that way.

File: code/modules/cargo/packs.dm [hunk 12]
```diff
@@ -2412,6 +2412,22 @@
 	crate_name = "deluxe keg"
 	crate_type = /obj/structure/closet/crate
 
+/datum/supply_pack/misc/paper_work
+	name = "Freelance Paper work"
+	desc = "The Nanotrasen Primary Bureaucratic Database Intelligence (PDBI) reports that the station has not completed its funding and grant paperwork this solar cycle. In order to gain further funding, your station is required to fill out (10) ten of these forms or no additional capital will be disbursed. We have sent you ten copies of the following form and we expect every one to be up to Nanotrasen Standards." // Disbursement. It's not a typo, look it up.
```
Comment (deathride58) [inline code/modules/cargo/packs.dm:2417]: According to the description, there's supposed to be 10, but there's only actually nine in the crate

File: code/modules/crafting/recipes.dm (hunk 7 above)
Comment (deathride58) [inline code/modules/crafting/recipes.dm]: Requiring fountain pens specifically is a little silly
```suggestion
	reqs = list(/obj/item/pen = 1,
```

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Review by deathride58 — APPROVED** [review]

**Pull request merged**